apps/new-tab/src/App.tsx

```
@@ -619,61 +683,84 @@
-                      class={`${uuidv4()} slot group h-fit`}
-                      data-swapy-slot={item}
-                    >
+                  filteredWidgets().map((item: any) => {
+                    console.log(item);
+                    return (
                       <div
-                        class="widget group"
-                        data-swapy-item={widgetOrder()[item]}
+                        class={`${uuidv4()} slot group h-fit`}
+                        data-swapy-slot={item}
                       >
```

Biome (1.9.4)

[error] Missing key property for this element in iterable.

The order of the items may change, and having a key can help React identify which item was moved.
Check the React documentation. 

(lint/correctness/useJsxKeyInIterable)